metapkg/commands/copytree.py

```diff
@@ -0,0 +1,143 @@
+from __future__ import annotations
+from typing import *
+
+import logging
+import os
+import pathlib
+import shutil
+import stat
+
+from . import base
+
+
+logger = logging.getLogger('metapkg.copytree')
+
+
+class CopyTree(base.Command):
```

Comment:
This should be in the form of a helper script like `targets/_helpers/trim-install.py` because it needs to be callable from the generated package build script, and `metapkg` itself wouldn't be (think source RPM).  See how `trim-install` is used for an example.

Reply:
kk, will make this in a separate PR for clarity.